bigbluebutton-html5/imports/api/captions/server/methods/appendText.js

```diff
@@ -8,9 +8,7 @@ import {
   appendTextURL,
 } from '/imports/api/note/server/helpers';
 
-export default function appendText(body, text, locale) {
-  const { meetingId } = body;
-
+export default function appendText(text, locale, meetingId) {
   check(meetingId, String);
```

Comment:
Can you please get meetingId from 

  const { meetingId } = extractCredentials(this.userId);

instead of passing from client?